plugins/lsp_java.lua

```diff
@@ -0,0 +1,43 @@
+-- mod-version:3 -- lite-xl 2.1
+
+local common = require "core.common"
+local config = require "core.config"
+local lsp = require "plugins.lsp"
+
+local installed_path_plugin = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_java"
+local installed_path_library = USERDIR .. PATHSEP .. "libraries" .. PATHSEP .. "jdk"
```

Comment:
This should probably be returned by a `jdk` `library`, kind of like what `nodejs` is doing.

Reply:
I kind of missed that part with nodejs.lua, how does it work?